src/components/fab.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Image, StyleSheet, TouchableOpacity} from 'react-native';
+import {NavigationComponentProps} from 'react-native-navigation';
+import {goToAddUser} from '../utils/navigation';
+
+export const AddUserButton = (props: NavigationComponentProps) => {
```

Comment:
uma coisa legal é sempre manter o componente com o mesmo nome do arquivo. nesse caso, seria algo assim
```suggestion
// src/components/add-user-fab.tsx
export const AddUserFAB: React.FC<AddUserFABProps> = (props) => {
```